pkg/dnsbl/dnsbl.go

```diff
@@ -38,7 +37,7 @@ func (c *Checker) Query(ipAddress string, lists io.Reader) {
 	go func() {
 		for response := range responses {
 			if response > 0 {
-				c.Positive += response
+				c.positive += response
```

Comment:
As discussed with @miguelbernadi, `response` is a count which may be `> 1`. In the very end of this commit, the number of pages queried gets increased by 1 only, so that the score `positive/queried` might be bigger than 100% and become completely pointless. I'd rather do a `+= 1` over here.

Reply:
But then we'd know the number of providers that have a list in which we appear, not the number of lists we appear in. 

The current approach over represents our presence in blacklists as a conscious trade-off to limit complexity. Even though the value is not really a percentage (we could call it a score to prevent that) our goal is always to keep it as low as possible. There's a limit set only for convenience due to the reporting model required by Nagios checks (which ZenOSS uses).